resources/testharness.js

```diff
@@ -2258,12 +2258,28 @@ policies and contribution forms [3].
         }
         var node = output_document.getElementById("log");
         if (!node) {
-            if (!document.body || document.readyState == "loading") {
+            if (!document.readyState == "loading") {
                 return;
             }
-            node = output_document.createElement("div");
+            node = output_document.createElementNS("http://www.w3.org/1999/xhtml", "div");
             node.id = "log";
-            output_document.body.appendChild(node);
+            if (output_document.body) {
+                output_document.body.appendChild(node);
+            } else {
+                var is_svg = false;
+                try {
+                    is_svg = output_document.documentElement instanceof SVGSVGElement;
```

Comment:
This doesn't work if `output_document != document`, right?

Reply:
@jgraham oh that's true

Reply:
@jgraham fixed